docs/pages/api/renderers/web.md

```diff
@@ -2,6 +2,7 @@ import Docs from "@site/pages/api/generated/renderer-WebRenderer.md"
 
 # Web Renderer
 
-Represents an instance of a website opened with Chromimum embeded inside the compositor. Used by a [`WebView` component](../components/WebView). Only one `WebView` component can use a specific instance at a time.
+Represents an instance of a website opened with Chromium embeded inside the compositor. Used by a [`WebView` component](../components/WebView). Only one `WebView` component can use a specific instance at a time.
+Before the web renderer can be used, you need to make sure that compositor with web rendering support is used.
```

Comment:
```suggestion
Represents an instance of a website opened with Chromium embedded inside the compositor. Used by a [`WebView` component](../components/WebView). Only one `WebView` component can use a specific instance at a time.
Before the web renderer can be used, you need to make sure that compositor with web rendering support is used.
```

Reply:
Try using some typo checker in IDE / editor

Reply:
I used grammarly. Weird that it didn't pick that up